README.md

```diff
@@ -902,6 +904,8 @@ _For a more comprehensive/advanced/better categorized/... list of Linux audio so
 
 - [Clipgrab](https://clipgrab.org/) - A friendly downloader for YouTube and other sites.
 - [![Open-Source Software][oss icon]](https://github.com/NickvisionApps/Parabolic) [Parabolic](https://github.com/NickvisionApps/Parabolic) - Download web video and audio.
+- [![Open-Source Software][oss icon]](https://github.com/spotDL/spotify-downloader) [spotDL](https://github.com/spotDL/spotify-downloader) - Download your Spotify playlists and songs along with album art and metadata (from YouTube if a match is found).
+- [![Open-Source Software][oss icon]](https://github.com/Shabinder/SpotiFlyer) [SpotiFlyer](https://github.com/Shabinder/SpotiFlyer) - Kotlin Multiplatform Music Downloader, Supports Spotify / Gaana / Youtube Music / Jio Saavn / SoundCloud. 
```

Comment:
From Github's README:
Please refer above for latest updates.

- Spotiflyer itself won't be getting any more updates, We all thank you for your support and hope to see you in Soundbound Telegram Community.

Reply:
Thanks for catching that! Resolved :-)